src/components/WebexData/WebexData.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const AdapterContext = React.createContext();
+
+export default function WebexData(props) {
+  return <AdapterContext.Provider value={props.adapter}>{props.children}</AdapterContext.Provider>;
+}
+
+WebexData.propTypes = {
+  adapter: PropTypes.exact({
+    roomsAdapter: PropTypes.object.isRequired,
+    activitiesAdapter: PropTypes.object.isRequired,
+    peopleAdapter: PropTypes.object.isRequired,
+  }).isRequired,
+  children: PropTypes.node.isRequired,
```

Comment:
No trailing comma, please
```suggestion
  children: PropTypes.node.isRequired
```

Reply:
see my other comments